tasks/rollup.js

```diff
@@ -78,7 +78,7 @@ module.exports = function(grunt) {
         preferConst: options.preferConst,
         pureExternalModules: options.pureExternalModules,
         treeshake: options.treeshake,
-        interop: options.interop
+        output: { interop: options.interop }
```

Comment:
Wouldn't it be more clear to put it on the options object passed to `bundle.generate` without the "output" object wrapper.